### PR TITLE
[2.20.x] DDF-5877 Make geo attributes indexed in solr

### PR DESCRIPTION
**Pull request by rececoffin**
#### What does this PR do?
Indexes geo fields to improve search experience when searching on 
#### Who is reviewing it? 

@bellcc 
@derekwilhelm 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@jlcsmith
@vinamartin
-->

#### How should this be tested?

Upload data that has geo location associated with it. Run a search for `location is empty` and ensure that the data does not come back. Then change the query to `location is NOT empty` and make sure the expected data comes back.

#### What are the relevant tickets?
Fixes: #5877 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/plugin/catalog-plugin-videothumbnail/pom.xml [hunk 1]
```diff
@@ -123,7 +123,7 @@
                         <configuration>
                             <rules>
                                 <ArtifactSizeEnforcerRule implementation="org.codice.maven.artifactsize.ArtifactSizeEnforcerRule">
-                                    <maxArtifactSize>26.4_MB</maxArtifactSize>
+                                    <maxArtifactSize>56.4_MB</maxArtifactSize>
```
Comment (andrewkfiedler) [inline catalog/plugin/catalog-plugin-videothumbnail/pom.xml]: Is this a separate change we need?

Comment (rececoffin) [reply catalog/plugin/catalog-plugin-videothumbnail/pom.xml]: I didn't know if it needed done separately, I just added it here because it is an easy fix.

**Comment by jlcsmith** [comment]
Important note - this fix incorporates a Solr schema change.  Any data will need to be re-indexed to take advantage of the feature.

File: catalog/plugin/catalog-plugin-videothumbnail/pom.xml (hunk 1 above)
Comment (pklinef) [reply catalog/plugin/catalog-plugin-videothumbnail/pom.xml]: This looks unrelated to this PR.

**Comment by rececoffin** [comment]
build now

**Pull request merged**